src/AdvApi32/AdvApi32+ServiceType.cs

```diff
@@ -59,7 +59,12 @@ public enum ServiceType : uint
             /// <summary>
             /// The service can interact with the desktop.
             /// </summary>
-            SERVICE_INTERACTIVE_PROCESS = 0x00000100
+            SERVICE_INTERACTIVE_PROCESS = 0x00000100,
+            
+            /// <summary>
+            /// The service current type is not changed.
+            /// </summary>
+            SERVICE_NO_CHANGE = 0xFFFFFFFF
```

Comment:
Can you also add a trailing comma to this last enum, so that the next time an enum value is added, it doesn't have to change this line?